frontend/src/app/features/calendar/op-calendar.service.ts

```diff
@@ -34,11 +34,13 @@ export class OpCalendarService extends UntilDestroyedMixin {
   }
 
   applyNonWorkingDay({ date }:{ date?:Date }, nonWorkingDays:IDay[]):string[] {
-    const formatted = moment(date).format('YYYY-MM-DD');
-    if (date && (this.weekdayService.isNonWorkingDay(date) || nonWorkingDays.find((el) => el.date === formatted))) {
-      return ['fc-non-working-day'];
+    if (date) {
+      const utcDate = new Date(date.getTime() + date.getTimezoneOffset() * 60000);
```

Comment:
Could you make the magic number more clear, perhaps explain in a comment or the commit why this is needed